### ISSUE TITLE
Accordion: navigation within the header is broken.

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#672]
It seems that keyboard navigation (tab) within a custom accordion header is broken. For this reason tests are failing.

**Comment by wkeese** [comment]
Regression fixed by ibm-js/delite@e232bc563e5578b79577532921cf4ee1bf04c0ce.
